summingbird-core/src/main/scala/com/twitter/summingbird/Dependants.scala

```diff
@@ -42,14 +42,16 @@ case class Dependants[P <: Platform[P]](tail: Producer[P, Any]) extends Dependan
   }
 
   /**
-   * Return the first dependants of this node AFTER merges. This will include
-   * no MergedProducer nodes, but all parents of the returned nodes will be Merged
-   * or the argument.
+   * Return the first dependants of this node AFTER merges and skipping Alsos.
+   * This will include no MergedProducer or AlsoProducer nodes, but all dependencies
```

Comment:
no, I think that is right. It is saying if you look up (towards dependencies) for all the returned nodes, they will either be Merged, Also or inp. I could add a test to verify that. In fact I should (but the test I added was related).